### ISSUE TITLE
Regarding the connection issue of processed data

### ISSUE THREAD

**Issue by ruiqiyan** [shreyesss/HDRSplat#1]
It seems that the processed data cannot be accessed

**Comment by shreyesss** [comment]
Hey, we are still working on uploading the fully processed data. What you can do is download the original RawNerf data and process using the methodology already detailed.

**Comment by ruiqiyan** [comment]
Thank you for your reply. Sorry to bother you, may I ask if this method can handle Gaussian rendering of overexposed images in addition to solving underexposed images.

**Comment by shreyesss** [comment]
Yes! The method can reconstruct scenes from raw images which are either under or over exposed, the parameters of the post-processing pipeline can be adjusted accordingly  when converting the novel raw views to LDR.

**Comment by ruiqiyan** [comment]
Thank you for your reply. It seems that the raw RawNerf data is also inaccessible. May I ask what files are in the raw folder of the data. If I use my own data, what else do I need for the raw data besides the Spark and Images folders?

**Comment by ruiqiyan** [comment]
Thank you for your reply. It seems that the raw RawNerf data is also inaccessible. May I ask what files are in the raw folder of the data. If I use my own data, what else do I need for the raw data besides the two folders sparse and images?

**Comment by shreyesss** [comment]
Rawnerf data can be downloaded from here https://bmild.github.io/rawnerf/
For your own dataset, the you’ll need to extract 
these 5 metadata values from your raw file for processing 
ISO,
Exposure
BlackLevel
WhiteLevel
Cam2RGB

**Comment by ruiqiyan** [comment]
I think you misunderstood my question. What files should I need initially if I want to process my own dataset. Do you need three folders like RawNeRF: raw, sparse, images? If so, may I ask what is inside the raw folder. also https://bmild.github.io/rawnerf/ After accessing, the dataset cannot be downloaded.
